### PR TITLE
Fix syntax errors on osu!weekly #95–#99, 103

### PR DESCRIPTION
**Pull request by MilkyIQ**
Fixes the header syntax errors as described in [this message (Discord)](https://discord.com/channels/188630481301012481/218677502141399041/837767574161850378) as well as a forgotten forward slash on osu!weekly #103

Apologies in advance if I messed anything up, I'm not extremely well versed in HTML or CSS

### CODE REVIEW

**Comment by Joehuu** [comment]
Is there a reason not to convert these headings to markdown?

**Comment by MilkyIQ** [comment]
No there isn't. Probably should've done that in the first place

File: news/2017-03-09-osuweekly-95.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ tumblr_url: http://osunews.tumblr.com/post/158194347743/osuweekly-95
   <li><a href="#twc">Taiko World Cup</a></li>
 </ul>
 
-<h2 name="news" id="news">Weekly news</h2>
+<h2 name="news" id="news" class="osu-md__header osu-md__header--2">Weekly news</h2>
```
Comment (TPGPL) [inline news/2017-03-09-osuweekly-95.md:25]: couldnt it be 
d
```suggestion
## <a id="news"></a>Weekly news
```

**Comment by bdach** [comment]
There is a reason to not convert, and it's the table of contents in the aforementioned posts. Note that they use anchors, so the `id` in the `h2` tags is actually used to link to those headings.

As far as I can see osu-web does not autogenerate anchor names for markdown headings, so you'd have to remove the table of contents, if you want to convert, as it'd be useless.

**Comment by MilkyIQ** [comment]
If there's no advantage to changing to markdown headers, then should we just leave that job up to a different pr? There are quite a few other newsposts that use this same method, if I'm not mistaken

File: news/2017-03-17-osuweekly-96.md [hunk 2]
```diff
@@ -44,13 +45,13 @@ A new weekly, a slightly modified format! We have kind of already started rollin
...
 
-<h2 name="score" id="score">Scorewatch: March week 3 (With Scorewatch Patrol)</h2>
+<h2 name="score" id="score" class="osu-md__header osu-md__header--2">Scorewatch: March week 3 (With Scorewatch Patrol)</h2>
 
-[Ekoro](https://osu.ppy.sh/users/284905), currently participating in TWC 2017, is not just capable of whacking those drums. He pulled off a mindblowing EZFL 98.50% FC on [9mm Parabellum Bullet - Inferno](https://osu.ppy.sh/beatmaps/1028733) on The Eclipse difficulty, giving him 385pp. Ekoro has been super capable in playing FL, and this score is nothing short of breathtaking.
+[Ekoro](https://osu.ppy.sh/users/284905), currently participating in TWC 2017, is not just capable of whacking those drums. He pulled off a mindblowing EZFL 98.50% FC on [9mm Parabellum Bullet - Inferno](https://osu.ppy.sh/beatmapsets/482090) on The Eclipse difficulty, giving him 385pp. Ekoro has been super capable in playing FL, and this score is nothing short of breathtaking.
```
Comment (cl8n) [inline news/2017-03-17-osuweekly-96.md:54]:
```suggestion
[Ekoro](https://osu.ppy.sh/users/284905), currently participating in TWC 2017, is not just capable of whacking those drums. He pulled off a mindblowing EZFL 98.50% FC on [9mm Parabellum Bullet - Inferno](https://osu.ppy.sh/beatmapsets/482090#osu/1028733) on The Eclipse difficulty, giving him 385pp. Ekoro has been super capable in playing FL, and this score is nothing short of breathtaking.
```

File: news/2017-03-30-osuweekly-98.md [hunk 3]
```diff
@@ -22,35 +23,35 @@ This is the osu!weekly, coming to you straight from the new homepage! What new h
...
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/x0Nss79R0Js" frameborder="0" allowfullscreen></iframe>
 
-[WubWoofWolf](https://osu.ppy.sh/users/39828) makes the SS rank feel like a joke to him as he pulled that out of [Kuroneko Dungeon - Ryoushi no Umi no Lindwurm](https://osu.ppy.sh/beatmaps/941254) on yf & Crystal's Extra with HDHR to gain 312pp. No one in the leaderboards are remotely close to him. See this flawless play for yourself!
+[WubWoofWolf](https://osu.ppy.sh/beatmapsets/405167#osu/941254) makes the SS rank feel like a joke to him as he pulled that out of [Kuroneko Dungeon - Ryoushi no Umi no Lindwurm](https://osu.ppy.sh/beatmapsets/405167#osu/941254) on yf & Crystal's Extra with HDHR to gain 312pp. No one in the leaderboards are remotely close to him. See this flawless play for yourself!
```
Comment (cl8n) [inline news/2017-03-30-osuweekly-98.md:54]:
```suggestion
[WubWoofWolf](https://osu.ppy.sh/users/39828) makes the SS rank feel like a joke to him as he pulled that out of [Kuroneko Dungeon - Ryoushi no Umi no Lindwurm](https://osu.ppy.sh/beatmapsets/405167#osu/941254) on yf & Crystal's Extra with HDHR to gain 312pp. No one in the leaderboards are remotely close to him. See this flawless play for yourself!
```
copy and paste with care :^)

**Review by cl8n — CHANGES_REQUESTED** [review]
use @TPGPL's suggestion for the headers, that's the standard workaround for it used in other news posts. also adding classnames to html like this is not good as they could change any time on osu-web

**Comment by cl8n** [comment]
> As far as I can see osu-web does not autogenerate anchor names for markdown headings, so you'd have to remove the tables of contents, if you want to convert, as they'd be useless.

jsyk it's done on wiki but not news. I think it's because news has no contents sidebar

**Review by cl8n — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**